api/Animation.json

```diff
@@ -0,0 +1,945 @@
+{
+  "api": {
+    "Animation": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Animation",
+        "support": {
+          "webview_android": {
+            "version_added": "39",
```

Comment:
Rather than a note, we can describe this using"alternate_name

"webview_android": [
  {
     "version_added": "44"
  },
  {
      "alternate_name": "AnimationPlayer",
      "version_added": "39",
       "version_removed": "44"
  }
]
(indentation not necessarily correct)